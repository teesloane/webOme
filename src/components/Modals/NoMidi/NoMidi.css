@keyframes floatSynth{
 0% {
   transform: translateY(0);
 } 

 100% {
   transform: translateY(12px);
 }

}

.NoMidi {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.NoMidi-text {
  font-size: 1.6rem;
  margin: 2rem 0;
}



.NoMidi-image-synth {
  animation: floatSynth 1.5s ease infinite alternate;
}

.NoMidi-image-synth::after {
  width: 300px;
  height: 100px;
  box-shadow: 3px 3px 3px rgba(0, 0, 0, 0.5);
  background: black;
}