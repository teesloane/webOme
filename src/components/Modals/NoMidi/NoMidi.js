import React, { Component, PropTypes } from 'react';
import './NoMidi.css';

const NoMidi = props => {
  return (
    <div className="NoMidi">
      <div className="NoMidi-text">You don't have any midi devices! Sadly <em>WebOme</em> cannot function without some cool doo-dad midi devices along-side it.</div> 
      <div className="NoMidi-image-carousel" />
    </div>
  );
};

export default NoMidi;