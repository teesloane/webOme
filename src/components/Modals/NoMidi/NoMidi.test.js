import NoMidi from './NoMidi'

it('matches snapshot', () => {
  const component = shallow(<NoMidi />);
  expect(component).toMatchSnapshot();
})