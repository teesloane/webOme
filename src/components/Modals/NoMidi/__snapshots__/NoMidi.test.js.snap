exports[`test matches snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="NoMidi">
    <div
        className="NoMidi-text">
        You don\'t have any midi devices! Sadly 
        <em>
            WebOme
        </em>
         cannot function without some cool doo-dad midi devices along-side it.
    </div>
    <img
        alt=""
        className="NoMidi-image-synth"
        src="test-file-stub" />
</div>,
  "nodes": Array [
    <div
      className="NoMidi">
      <div
            className="NoMidi-text">
            You don\'t have any midi devices! Sadly 
            <em>
                  WebOme
            </em>
             cannot function without some cool doo-dad midi devices along-side it.
      </div>
      <img
            alt=""
            className="NoMidi-image-synth"
            src="test-file-stub" />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Unknown />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="NoMidi">
          <div
                    className="NoMidi-text">
                    You don\'t have any midi devices! Sadly 
                    <em>
                              WebOme
                    </em>
                     cannot function without some cool doo-dad midi devices along-side it.
          </div>
          <img
                    alt=""
                    className="NoMidi-image-synth"
                    src="test-file-stub" />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="NoMidi">
          <div
                    className="NoMidi-text">
                    You don\'t have any midi devices! Sadly 
                    <em>
                              WebOme
                    </em>
                     cannot function without some cool doo-dad midi devices along-side it.
          </div>
          <img
                    alt=""
                    className="NoMidi-image-synth"
                    src="test-file-stub" />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Unknown />,
}
`;
